packages/glimmer-runtime/lib/syntax/core.ts

```diff
@@ -241,16 +247,19 @@ export class Modifier implements StatementSyntax {
   }
 
   constructor(options) {
+    super();
     this.path = options.path;
-    this.params = options.params;
-    this.hash = options.hash;
+    this.args = options.args;
   }
 
-  evaluate(stack) {
-    return stack.createMorph(Modifier);
+  compile(compiler: CompileInto, env: Environment) {
+    compiler.append(new ModifierOpcode({ name: intern(this.path) }));
+  }
+
+  prettyPrint(): PrettyPrint {
+    return new PrettyPrint('modifier', this.path, [this.args.prettyPrint()]);
   }
```

Comment:
`prettyPrint` is legacy, 🔥. It was added for the old version of the visualizer but is now completely replaced by `toJSON`. If something else is giving you a `TypeError` for not having it, you can probably do a sweep in the codebase to remove all of them